### DISCUSSION TITLE
Call for Feedback: Adding Stepper Layout to Add to Cart with Options Block

### DISCUSSION THREAD

**Discussion by shaybanshee** [woocommerce/woocommerce#53039] [WooCommerce, Blocks, and Site Editing]
Hello! This discussion relates to the call for feedback post [here](https://developer.woocommerce.com/2024/11/20/call-for-feedback-stepper-layout-in-add-to-cart-block/).

@woocommerce/woo-fse and @gigitux have been working on a change to the CSS of the classic Add to Cart with Options experience, adding a much requested `Stepper` to be able to add and remove multiples of the same product to a cart in one fell swoop. We have requested feedback from the community and added it as an experimental feature flag starting in WooCommerce 9.5 Beta. Please add any feedback here after testing, or open an issue - feel free to link this discussion as well so we can keep all our feedback connected.

Thanks, and happy testing!

**Comment by ralucaStan** [comment]
We should try out the stepper in the Cart checkout block.

**Comment by medavidallsop** [comment]
I posted this issue before this discussion was created, please consider this in your feedback: https://github.com/woocommerce/woocommerce/issues/53031

Thank you,

**Reply by gigitux** [reply]
Thanks! I replied on the issue! Please let us know if something isn't clear, or you think that we should improve! 🙇

**Comment by KoolPal** [comment]
Suggestion - for products marked as sold individually PLEASE DO **NOT** SHOW THE QUANTITY SELECTOR 
Thanks

**Reply by gigitux** [reply]
Thanks for the feedback! Could you share reproduction steps? I'm not able to reproduce this issue:

https://github.com/user-attachments/assets/62155894-cdfd-4eed-9f55-378493bc730f

**Comment by gigitux** [comment]
With https://github.com/woocommerce/woocommerce/pull/54116, the Stepper Layout will be available in WooCommerce 9.7. Thank you for your valuable feedback! This discussion will now be closed, but feel free to add a comment or open a dedicated issue for any further concerns or feedback.
